### ISSUE TITLE
Request for Icon

### ISSUE THREAD

**Issue by seatechdev** [AndreLGava/font-awesome-extension#4]
Online or Mobile Wallet

**Comment by AndreLZGava** [comment]
Do you have an image to illustrate how you would like this?

**Comment by seatechdev** [comment]
No.  May be a mobile phone with cards or money in it?  Please see the google search results at the URL below:

https://www.google.com/search?q=mobile+wallet+icon&espv=2&biw=1218&bih=632&tbm=isch&tbo=u&source=univ&sa=X&ved=0ahUKEwiiwLC98P_MAhVK7WMKHd8PCUsQsAQIJw&dpr=1
